tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	parser "github.com/kubevirt/monitoring/pkg/metrics/parser"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+// excludedMetrics defines the metrics to ignore,
+// open bug: https://bugzilla.redhat.com/show_bug.cgi?id=2219763
+// Do not add metrics to this list!
+var excludedMetrics = map[string]struct{}{
```

Comment:
Can't you fix the names as part of this PR to satisfy the linter instead of ignoring the wrong names?

Reply:
@sradco, @avlitman WDYT? can we just fix the names within this PR?

Reply:
@assafad From my side I think it make sense to fix in this PR

Reply:
Yes, I will probably add identical recording rules with correct names, and leave the wrong ones, until we deprecated them in a future version.